commands/new_function.go

```diff
@@ -24,6 +24,7 @@ var (
 )
 
 func init() {
+	newFunctionCmd.Flags().StringVarP(&apiVersion, "version", "v", defaultAPIVersion, "The version to use for YAML stack file")
```

Comment:
rather than apiVersion, schemaVersion would be more apt. apiVersion is a Kubernetes thing.